x/storage/keeper/cross_app_group.go

```diff
@@ -374,6 +377,94 @@ func (app *GroupApp) handleUpdateGroupMemberSynPackage(ctx sdk.Context, header *
 	}
 }
 
+func (app *GroupApp) handleUpdateGroupMemberV2SynPackage(ctx sdk.Context, header *sdk.CrossChainAppContext, updateGroupPackage *types.UpdateGroupMemberV2SynPackage) sdk.ExecuteResult {
+	err := updateGroupPackage.ValidateBasic()
+	if err != nil {
+		return sdk.ExecuteResult{
+			Payload: types.UpdateGroupMemberAckPackage{
+				Status:    types.StatusFail,
+				Operator:  updateGroupPackage.Operator,
+				ExtraData: updateGroupPackage.ExtraData,
+			}.MustSerialize(),
+			Err: err,
+		}
+	}
+
+	groupInfo, found := app.storageKeeper.GetGroupInfoById(ctx, math.NewUintFromBigInt(updateGroupPackage.GroupId))
+	if !found {
+		return sdk.ExecuteResult{
+			Payload: types.UpdateGroupMemberAckPackage{
+				Status:    types.StatusFail,
+				Operator:  updateGroupPackage.Operator,
+				ExtraData: updateGroupPackage.ExtraData,
+			}.MustSerialize(),
+			Err: types.ErrNoSuchGroup,
+		}
+	}
+
+	switch updateGroupPackage.OperationType {
+	case types.OperationAddGroupMember, types.OperationDeleteGroupMember:
+		err = app.handleAddOrDeleteGroupMemberOperation(ctx, groupInfo, updateGroupPackage)
```

Comment:
if it is add member, it is possible to have expire time, but can not find it here now.

Reply:
resolved, allow creation group Member with expiration from cross-chain package now